website/cue/reference/releases/0.28.1.cue

```diff
@@ -0,0 +1,43 @@
+package metadata
+
+releases: "0.28.1": {
+	date:     "2023-03-03"
+	codename: ""
+
+	description: """
+		This patch release contains a few fixes for regressions in 0.28.1.
+
+		**Note:** Please see the release notes for [`v0.28.0`](/releases/0.28.0/) for additional changes if upgrading from
+		`v0.27.X`. In particular, the upgrade guide for breaking changes.
+		"""
+
+	changelog: [
+		{
+			type: "fix"
+			scopes: ["aws provider"]
+			description: """
+				All AWS components now correctly sign requests again. This impacted all AWS
+				components except the `aws_s3` sink and caused the components to output AWS service
+				errors for each request.
+				"""
+			pr_numbers: [16651]
+		},
+		{
+			type: "fix"
+			scopes: ["socket source"]
+			description: """
+				The `framing.*.max_length` configuration options can again be used on the `socket`
+				source. Previously they returned an error that they conflicted with the deprecated
+				top-level `max_length` configuration option.
+				"""
+			pr_numbers: [16631]
+		},
+	]
+
+	commits: [
+		{sha: "764c542e36a13d2846c01a2d64b6a2d033fa7797", date: "2023-03-01 15:34:20 UTC", description: "Add known issues for 0.28.0.", pr_number:                                    16649, scopes: ["releasing"], type:     "chore", breaking_change: false, author: "Jesse Szwedko", files_count:   1, insertions_count:  12, deletions_count:  0},
+		{sha: "366d34eb93e5852350b6675d96a2701d5d4611e1", date: "2023-03-01 16:22:29 UTC", description: "Add a links to footer for to the /releases and /downloads pages", pr_number: 16441, scopes: ["website"], type:       "chore", breaking_change: false, author: "neuronull", files_count:       3, insertions_count:  16, deletions_count:  1},
```

Comment:
Hmmm I could be missing it but I'm not seeing this commit ? It actually looks like others listed here might also be mssing ?

Reply:
Good observation. The ones you don't see on this PR are already on `v0.28` compared to the last release:

```
git log v0.28.0...v0.28 | cat
366d34eb9 - (origin/v0.28, v0.28) chore(website): Add a links to footer for to the /releases and /downloads pages (#16441) (2 days ago) <neuronull>
764c542e3 - chore(releasing): Add known issues for 0.28.0. (#16649) (2 days ago) <Jesse Szwedko>
```

These ones were cherry-picked onto the release branch to be included in the documentation early

Reply:
It may have been cherry-picked into the `0.28` branch already

Reply:
Indeed: https://github.com/vectordotdev/vector/commit/366d34eb93e5852350b6675d96a2701d5d4611e1